### ISSUE TITLE
Add stats second level model

### ISSUE THREAD

**Issue by rob-luke** [mne-tools/mne-nirs#77]
* https://github.com/nilearn/nilearn/issues/2372
* https://github.com/nilearn/nilearn/blob/master/nilearn/stats/second_level_model/second_level_model.py
* https://www.statsmodels.org/stable/generated/statsmodels.genmod.generalized_linear_model.GLM.html

Require:
* Dataset
* Add tutorial

**Comment by rob-luke** [comment]
This is a first attempt

```python
from statsmodels.formula.api import rlm
rlm_model = rlm('theta ~ -1  + condition:Chroma:ch_name', input, groups=input["ID"]).fit()
print(rlm_model.summary())
```

**Comment by rob-luke** [comment]
Only the snipper from above is required. So perhaps a simple example will be sufficient, and do not add a function.

**Comment by rob-luke** [comment]
Closed in #167
